### ISSUE TITLE
when release the code?

### ISSUE THREAD

**Issue by 10183308** [HannaRiver/Pixel-Anchor#1]
when release the code?

thank you very much

**Comment by ghost** [comment]
@HannaRiver Are you planning to release Pixel-Anchor code?

**Comment by HannaRiver** [comment]
> @HannaRiver Are you planning to release Pixel-Anchor code?

yep. I am doing some optimization of CRPN, and I will reproduce this paper later.

**Comment by WeihongM** [comment]
hello,  Have you finished your work? @HannaRiver

**Comment by alanMachineLeraning** [comment]
hello, Have you finished your work? @HannaRiver

**Comment by ghost** [comment]
@HannaRiver any updates?

**Comment by zlotus** [comment]
Looking forward to your update 😊
